executor/show.go

```diff
@@ -173,6 +173,9 @@ func (e *ShowExec) fetchAll() error {
 		return e.fetchShowPrivileges()
 	case ast.ShowBindings:
 		return e.fetchShowBind()
+	case ast.ShowAnalyzeStatus:
+		e.fetchShowAnalyzeStatus()
```

Comment:
It's better to sort the result of analyze status on `Start_time` in descending order to let the latest records appear on the bottom of the result. Take this as an example:
<img width="963" alt="Screen Shot 2019-04-24 at 00 27 16" src="https://user-images.githubusercontent.com/5268763/56598774-c10b1f00-6627-11e9-95fb-ad232b54d1ba.png">

As you can see in the picture, there are a lot of history informations which can not be printed in a screen. If users want to take a look at the latest analyze jobs, they have to scroll the screen up. Which I think is somehow inconvenient.

In order to see the running jobs, we have to use `show analyze status where state = "running";` at present. If we sort the result in the descending order of `start_time`, the last records are running analyze jobs if there are. Thus the command can be simplified to `show analyze status`

Reply:
Some tables have fewer rows, thus newer start time does not mean they are running. Instead, the result is already sorted by their status update time, see  https://github.com/pingcap/tidb/pull/10172/files#diff-318b1ef9098a111a1e7f8b3784db7fb2R89

Reply:
OK. We should use update time indeed. How about sort the result in the descending order of `update_time`?

Reply:
I think it already sorts the result in the descending order of update_time at https://github.com/pingcap/tidb/pull/10172/files#diff-318b1ef9098a111a1e7f8b3784db7fb2R89.

Reply:
:joy: I wish the latest record appears in the last, it should be ascending order.